README.md

```diff
@@ -199,3 +207,40 @@ windows:
     - run: perl -Mabbreviation -e1
     - run: perl -MACH -e1
 ```
+
+### Using install-with-cpanm for installation in local library
+
+```yaml
+  local_lib:
+    runs-on: ubuntu-latest
+    name: 'local-lib'
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: 'install-with-cpanm'
+        uses: ./
+        with:
+          install: 'Simple::Accessor'
+          local-lib: '~/perl5'
+          sudo: false
+          args: '-v'
+```
+
+`-v` is for verbosity
+
+
+## License
```

Comment:
I added this license for clarity, since I was forking. You might want either to delete it, or to expand collaborator names from nicks.

Reply:
indeed we might prefer a more generic Perl'ish license

Reply:
Then you will need to change the short license name in package.json, assuming that artistic-2.0 is compatible with it. I guess ISC (the one here, which I just transcribed, keeping it in `package.json`) is the license in the boilerplate originally used, but it's pretty permissive, so I guess it's not a problem to change it. IIRC, the artistic will need to add a `LICENSE` file too here.